PWGCF/MultiparticleCorrelations/Core/MuPa-MemberFunctions.h

```diff
@@ -2080,7 +2105,7 @@ void BookParticleCutsHistograms()
   if (pc.fUseParticleCuts[ePtDependentDCAxyParameterization]) {
     pc.fPtDependentDCAxyFormula = new TFormula("fPtDependentDCAxyFormula", pc.fsParticleCuts[ePtDependentDCAxyParameterization].Data());
     // As a quick insanity check, try immediately to evaluate something from this formula:
-    if (std::isnan(pc.fPtDependentDCAxyFormula->Eval(1.44))) {
+    if (isnan(pc.fPtDependentDCAxyFormula->Eval(1.44))) {
```

Comment:
Any reason for removing the `std::` on isnan?
I think this was introduced directly on gitHub to standardize the code and perhaps you are still using an old local version. Please, check it

Reply:
I didn't pull these changes in my class done directly on GitHub, that's the only reason. In the latest commit, I switched back to std::